pkg/secrets/secrets.go

```diff
@@ -0,0 +1,29 @@
+package secrets
+
+import (
+	"strings"
+)
+
+type Decrypter interface {
+	Decrypt() ([]byte, error)
```

Comment:
Why byte array instead of string?

Reply:
It's implemented this way on the java side because we ran into issues with binary java keystore. Right now dinghy and terraformer don't need to use a byte array but just in case we start using secrets that aren't basic strings...

Reply:
The only case where this matters (in Go) is if this thing return truly binary data.  I don't think that's likely.  If you're unsure of the type, then `interface{}` would be a better choice.  However, since we're in the context of a config, it will almost always be string.  I would prefer these functions return string since we're already unmarshalling in the library.